lib/matplotlib/lines.py

```diff
@@ -1362,9 +1357,7 @@ def set_dash_joinstyle(self, s):
             For examples see :doc:`/gallery/lines_bars_and_markers/joinstyle`.
         """
         s = s.lower()
-        if s not in self.validJoin:
-            raise ValueError('set_dash_joinstyle passed "%s";\n' % (s,)
-                             + 'valid joinstyles are %s' % (self.validJoin,))
+        cbook._check_in_list(self.validJoin, s=s)
```

Comment:
In cases below you don't use the parameter name but the function name, e.g.

~~~
    def set_default_alignment(self, d):
        if d not in ["left", "right", "top", "bottom"]:
        cbook._check_in_list(["left", "right", "top", "bottom"],
            raise ValueError(
                             default_alignment=d)
~~~

`default_alignment=d` vs. `s=s`. Not terribly important but it would be good to decide on one way or  the other and stick to it.

Reply:
I decided to always go with the parameter name.  (Neither solution is always great, but heh, you can see the function name in the traceback anyways.)